scripts/plugins/storage/sqlite/downgrade/11.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TABLE foglamp.destinations (
+    id            INTEGER                     PRIMARY KEY AUTOINCREMENT,                  -- Sequence ID
+    type          smallint                    NOT NULL DEFAULT 1,                         -- Enum : 1: OMF, 2: Elasticsearch
+    description   character varying(255)      NOT NULL DEFAULT '',                        -- A brief description of the destination entry
+    properties    JSON                        NOT NULL DEFAULT '{ "streaming" : "all" }', -- A generic set of properties
+    active_window JSON                        NOT NULL DEFAULT '[ "always" ]',            -- The window of operations
+    active        boolean                     NOT NULL DEFAULT 't',                       -- When false, all streams to this destination stop and are inactive
+    ts            DATETIME DEFAULT (STRFTIME('%Y-%m-%d %H:%M:%f', 'NOW', 'localtime')));         -- Creation or last update
+
+INSERT INTO foglamp.destinations ( id, description )
```

Comment:
why do we need this?

Reply:
the foglamp.streams has a FK constrains when we downgrade :
```
...
    CONSTRAINT streams_fk1 FOREIGN KEY (destination_id)
    REFERENCES destinations (id) MATCH SIMPLE
...
```
so we need to create a dummy row in the destinations table to handle the reference for the rows that are already in the streams before the downgrade.